worker/exec/konnector_test.go

```diff
@@ -410,6 +431,20 @@ echo "{\"type\": \"toto\", \"message\": \"COZY_URL=${COZY_URL}\"}"
 		var updatedAcc account.Account
 		err = couchdb.GetDoc(inst, consts.Accounts, acc.ID(), &updatedAcc)
 		require.NoError(t, err)
-		assert.Equal(t, updatedAcc.DefaultFolderPath, "/Administrative/my-konnector-1/account-1")
+		assert.Equal(t, updatedAcc.DefaultFolderPath, "/Administrative/Trainline/account-1")
 	})
 }
+
+func loadLocale() error {
+	locale := consts.DefaultLocale
+	assetsPath := config.GetConfig().Assets
+	if assetsPath != "" {
+		pofile := path.Join("../..", assetsPath, "locales", locale+".po")
+		po, err := os.ReadFile(pofile)
+		if err != nil {
+			return fmt.Errorf("Can't load the po file for %s", locale)
+		}
```

Comment:
And then failing directly instead of returning an error. This would simplify a little the test readability and could allow you to use `t.Cleanup` if you need to clean some stuff